src/SFA.DAS.EAS.Support.Web/Views/Account/SubHeader.cshtml

```diff
@@ -5,12 +5,20 @@
 
 <div class="grid-row">
     <div class="column-full">
+
+        <h5 class="heading-small">
+            <a href="/" class="govuk-back-link">
+                Home
+            </a>
+        </h5>
+
         <h1 class="heading-large">
             @(Model?.DasAccountName)
             @if (!string.IsNullOrEmpty(Model?.PublicHashedAccountId))
             {
                 <span class="heading-secondary"> Account ID @(Model?.PublicHashedAccountId), created @Model.DateRegistered.ToString("dd/MM/yyyy")</span>
             }
+            <span class="heading-secondary"> Levy Status  @Model.ApprenticeshipEmployerType</span>
```

Comment:
I think the LEVY or NON-LEVY value is supposed to be in a tag:
https://design-system.service.gov.uk/components/tag/

as per the design on the ticket.

Support console uses the old gov-uk stylesheets however so will need to find the correct style to apply.